package.json

```diff
@@ -5,7 +5,9 @@
     "author": "Deriv",
     "dependencies": {
         "@artsy/fresnel": "^3.2.1",
+        "@deriv/analytics": "^1.0.2",
```

Comment:
This needs to be on version `^1.0.3` since the new methods was recently published from `1.0.3` 👀

Reply:
Thanks @adrienne-deriv The current published version seems to be 1.0.2:
https://www.npmjs.com/package/@deriv/analytics

<img width="587" alt="Screenshot 2023-08-04 at 08 39 46" src="https://github.com/binary-com/deriv-com/assets/129744061/30fe267e-71ee-40e4-b42a-47b3f27669b0">